README.md

```diff
@@ -0,0 +1 @@
+This seem to be a test repo ;)
```

Comment:
Thanks for the contribution! Much appreciated. 

I wonder if we also should add a big title like README?